admiral/pkg/controller/common/common.go

```diff
@@ -38,6 +42,7 @@ const (
 	RolloutPodHashLabel           = "rollouts-pod-template-hash"
 	RolloutActiveServiceSuffix	  = "active-service"
 	RolloutStableServiceSuffix	  = "stable-service"
+	EnvoyFilterSuffix             = "1.13"
```

Comment:
Why hardcode this, we are getting it from the input params right?

Reply:
removed